src/chainlit/frontend/src/components/organisms/playground/submitButton.tsx

```diff
@@ -42,29 +40,31 @@ export default function SubmitButton() {
         };
       });
 
-      await client.getCompletion(prompt, userEnv, controller, (done, token) => {
-        if (done) {
-          setCompletionController(undefined);
-          return;
-        }
-        setPlayground((old) => {
-          if (!old?.prompt) return old;
+      await ChainlitAPI.getCompletion(
+        prompt,
+        userEnv,
+        controller,
+        accessToken,
+        (done, token) => {
+          if (done) {
+            setCompletionController(undefined);
+            return;
+          }
+          setPlayground((old) => {
+            if (!old?.prompt) return old;
 
-          return {
-            ...old,
-            prompt: {
-              ...old.prompt!,
-              completion: (old.prompt?.completion || '') + token
-            }
-          };
-        });
-      });
+            return {
+              ...old,
+              prompt: {
+                ...old.prompt!,
+                completion: (old.prompt?.completion || '') + token
+              }
+            };
+          });
+        }
+      );
     } catch (err) {
       setCompletionController(undefined);
-
-      if (err instanceof ClientError) {
-        toast.error(err.toString());
```

Comment:
If a completion request to an LLM provider (in the prompt playground) fails, we will not display the error to the user?

Reply:
I've added a toast on error in api file, so it should be displayed